js/ui/overlay/utils.js

```diff
@@ -6,12 +6,17 @@ const WINDOW_HEIGHT_PERCENT = 0.9;
 
 export const getElementMaxHeightByWindow = ($element, startLocation) => {
     const window = getWindow();
```

Comment:
```suggestion
    const $window = $(getWindow());
```
and replace all `$(width)` with `$width`